src/test/java/jenkins/plugins/office365connector/helpers/HttpWorkerAnswer.java

```diff
@@ -0,0 +1,32 @@
+package jenkins.plugins.office365connector.helpers;
+
+import static org.powermock.api.mockito.PowerMockito.mock;
+
+import jenkins.plugins.office365connector.HttpWorker;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+/**
+ * @author Damian Szczepanik (damianszczepanik@github)
+ */
+public class HttpWorkerAnswer implements Answer<HttpWorker> {
```

Comment:
catches passed parameters so they can be taken later to validate (by assertions)